src/desktop/add_extensions.desktop

```diff
@@ -0,0 +1,15 @@
+[Desktop Entry]
+Type=Application
+Categories=Settings;System;Qt;X-SuSE-YaST;X-SuSE-YaST-Software;
+
+X-SuSE-YaST-Call=scc
+X-SuSE-YaST-Argument=select_extensions
+X-SuSE-YaST-RootOnly=yes
+X-SuSE-YaST-Group=Software
+
+Icon=yast-product-registration
+Exec=/sbin/yast2 scc select_extensions
+
+Name=Add System Extensions/Modules
+GenericName=Add System Extensions/Modules
```

Comment:
I'm not sure if using the slash is OK in the label. IIRC the old styleguide was against this and suggested using `and` or `or` words instead.

I have checked the other items and `and` is used several times, no item with slash. So I'd change it to be more consistent. (Fix all places, it's repeated several times in the file.)